api/package.json

```diff
@@ -1,11 +1,24 @@
 {
   "name": "lol-wrapper_api",
   "version": "0.0.1",
-  "main": "index.js",
+  "main": "dist/index.js",
   "license": "MIT",
   "scripts": {
-    "build": "exit 0",
-    "test": "exit 0",
-    "serve": "exit 0"
+    "build": "tsc -p tsconfig.json",
+    "serve": "ts-node ./scripts/server.ts",
+    "test": "exit 0"
+  },
+  "dependencies": {
+    "@aws-sdk/client-s3": "^3.321.1",
+    "@aws-sdk/s3-request-presigner": "^3.321.1",
+    "aws-sdk": "^2.1367.0",
```

Comment:
Were these installed for a reason, or just because we think we'll be using them in the future?

Reply:
So we can keep these for now. But it might be that node 18 lambdas come with those dependencies already. So basically we might be able to make them dev dependencies in the future. We can test at deploy time